modules/tinymce/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Line separator scrolling in floating toolbars. #TINY-8948
 - A double bottom border appeared on inline mode editor for the `tinymce-5` skin. #TINY-9108
 - The editor header showed up even with no menubar and toolbar configured. #TINY-8819
+- Inline text pattern does not trigger if it matches only the end but not the start. #TINY-8947
```

Comment:
We prefer to use past tense with changelog messages

```suggestion
- Inline text pattern no longer triggers if it matches only the end but not the start. #TINY-8947
```

Reply:
fixed